composer.json

```diff
@@ -50,6 +50,9 @@
             "dev-master": "2.1.x-dev",
             "dev-develop": "2.2.x-dev",
             "dev-release-3.0.0": "3.0.x-dev"
+        },
+        "zf": {
+            "config-provider": "Zend\\Expressive\\Router\\ConfigProvider"
```

Comment:
As in the previous review, I would consider changing namespace for the repository.